const anotherFunction = () => {
    return new Promise((resolve, reject) => {
        if (false) {
            resolve('Solomeo')
        }else {
            reject('no saliÃ³ solomeo');
        }
    })
}

anotherFunction()
    .then(Response => console.log(Response))
    .catch(err => console.log(err));